lib/prelude.dx

```diff
@@ -203,21 +203,22 @@ data Bool =
 
 def BToW8 (x : Bool) : Word8 = %dataConTag x
 
-def W8ToB (x : Word8) : Bool = %toEnum Bool x
+def unsafeW8ToB (x : Word8) : Bool = %toEnum Bool x
+def W8ToB (x : Word8) : Bool = unsafeW8ToB $ %igt x (IToW8 0)
```

Comment:
Honestly I think that we should onyl have `unsafeW8ToB`. If you want to define a different way to pack words into booleans you should do that in user code. The C-like assumption to treat 0 as false and everything else as true is a bit shady to me.

Reply:
After all it's just a matter to writing `x != 0` to get the same result in user code.

Reply:
Hmmm, good point.  I guess since there's no general way to cast these types to each other, I shouldn't have expected that to work.